src/plugins/kibana/public/settings/sections/indices/upload/directives/upload_wizard.js

```diff
@@ -0,0 +1,95 @@
+import modules from 'ui/modules';
+import template from 'plugins/kibana/settings/sections/indices/upload/directives/upload_wizard.html';
+import IngestProvider from 'ui/ingest';
+import 'plugins/kibana/settings/sections/indices/add_data_steps/pattern_review_step';
+import 'plugins/kibana/settings/sections/indices/add_data_steps/parse_csv_step';
+import 'plugins/kibana/settings/sections/indices/add_data_steps/pipeline_setup';
+import 'plugins/kibana/settings/sections/indices/add_data_steps/upload_data_step';
+
```

Comment:
Shouldn't this still import `import '../../styles/_add_data_wizard.less';` since the markup depends on it?